File: clients/python/examples/collections-insert-one-id-uuid.py
from astrapy import DataAPIClient
from astrapy.ids import UUID

# Get an existing collection
client = DataAPIClient()
database = client.get_database(
    "**API_ENDPOINT**", token="**APPLICATION_TOKEN**"
)
collection = database.get_collection("**COLLECTION_NAME**")

# Insert a document into the collection
result = collection.insert_one(
    {
        "_id": UUID("1ef2e42c-1fdb-6ad6-aae4-e84679831739"),
        "name": "Jane Doe",
    },
)

File: clients/typescript/examples/collections-insert-one-id-uuid.ts
import { DataAPIClient, UUID } from "@datastax/astra-db-ts";

// Get an existing collection
const client = new DataAPIClient();
const database = client.db("**API_ENDPOINT**", {
  token: "**APPLICATION_TOKEN**",
});
const collection = database.collection("**COLLECTION_NAME**");

// Insert a document into the collection
(async function () {
  const result = await collection.insertOne({
    _id: UUID.v7(),
    name: "Jane Doe",
  });
})();
